kubernetes/values.yaml

```diff
@@ -0,0 +1,25 @@
+# Default values for torchserve helm chart.
+
+torchserve_image: pytorch/torchserve:latest-gpu
+
+namespace: torchserve
+
+torchserve:
+  management_port: 8081
+  inference_port: 8080
+  pvd_mount: /home/model-server/shared/
+  n_gpu: 1
+  n_cpu: 1
+  memory_limit: 4Gi
+  memory_request: 1Gi
+
+deployment:
+  replicas: 3
+
+persitant_volume:
+  size: 1Gi
+
+efs:
+  region: us-west-2
+  file_system_dns_name: fs-4d0ed148.efs.us-west-2.amazonaws.com
```

Comment:
file system dns and file system id will be very specific to user's env and we can't have defaults for these

Reply:
Fixed